coffee/cse.py

```diff
@@ -206,11 +207,11 @@ def __init__(self, exprs, header, hoisted, decls):
 
     @property
     def type(self):
-        return self.exprs.values()[0].type
+        return list(itervalues(self.exprs))[0].type
```

Comment:
This implicitly (before it was also the case) assumes that self.exprs has one entry, no?